### PR TITLE
Update for PM 0.7.0

### PR DESCRIPTION
**Pull request by cheese3660**
This updates the patch files for support for 0.7.0 (changes : + to +:, and also makes use of unquoted strings)
Don't merge yet, as I haven't built a zip, but testing this in game seems to work

### CODE REVIEW

**Comment by Safarte** [comment]
Why are the patch file names all shuffled? e.g. `add_LSConsumer_module.patch` contains the stuff from `add_LS_resources.patch` etc...

**Comment by cheese3660** [comment]
Should be fixed, I was copying from VSCode and made the assumption that VSCode sorts files the same way github does (spoiler: it does not)

File: plugin_template/swinfo.json
Comment (Safarte) [inline plugin_template/swinfo.json:1]: Shouldn't the min Patch Manager version be bumped up to 0.7.0?

Comment (cheese3660) [reply plugin_template/swinfo.json:1]: Ah, yeah, forgot about that, thanks

**Comment by Safarte** [comment]
lgtm, will merge when a release build is available

**Pull request merged**